docs/recipes/integration-test/example-lh-auth.test.js

```diff
@@ -7,23 +7,22 @@
 /* eslint-disable new-cap */
 
 /**
- * @fileoverview Example Jest tests for demonstrating how to run Lighthouse on an authenticated
+ * @fileoverview Example Mocha tests for demonstrating how to run Lighthouse on an authenticated
  * page as integration tests. See docs/recipes/auth/README.md for more.
  */
 
 /** @typedef {import('lighthouse/types/lhr')} LH */
 
 const puppeteer = require('puppeteer');
 const lighthouse = require('lighthouse');
+const {expect} = require('expect');
```

Comment:
According to `docs/recipes/integration-test/.eslintrc.cjs` this should be a global right?